src/Cli/ConfigGenerator.cs

```diff
@@ -681,6 +686,99 @@ private static bool TryUpdateDepthLimit(
             return true;
         }
 
+        /// <summary>
+        /// Attempts to update the Config parameters in the GraphQL runtime settings based on the provided value.
```

Comment:
I imagine this method can be used for future updates to REST runtime settings as well. Hence the generic name makes sense. Lets update the summary description as well saying that at the moment only GraphQL settings can be configured.

Reply:
updated